### DISCUSSION TITLE
Integration with SOLID JS

### DISCUSSION THREAD

**Discussion by saroipardavila** [silviogutierrez/reactivated#281] [Ideas]
Hey! I'm very excited to use this project in a real application. Ever since I came across it and tried it for the first time I fell in love with the approach. I have also been working with `solid.js` lately and I have to say I like it much better than React. That's why I think it would be a GREAT idea to be able to integrate `reactivated` with `solid.js`. 

I will try to read through the code and understand how it works and how it can be adapted to work with this other framework. However, I'm not a senior developer by any means, therefore any guidance or help is more than welcome !

Thanks again for creating this amazing library :)

**Comment by silviogutierrez** [comment]
Hi there,

Glad you like the library. I'm not intimately familiar with SolidJS but I think this is a good place to start:

https://www.solidjs.com/guides/server
https://github.com/silviogutierrez/reactivated/blob/6922107ebb3aa3b0d74833dd73a780597a75c4fb/packages/reactivated/src/renderer.tsx

Then you can hydrate in `client/index.tsx` with SolidJS instead of React.

Hope that helps.
